databricks/README.md

```diff
@@ -14,91 +14,103 @@ Monitor Databricks Spark applications with the [Datadog Spark integration][3]. I
 
 Configure the Spark integration to monitor your Apache Spark Cluster on Databricks and collect system and Spark metrics.
 
-1. Determine the best init script below for your Databricks cluster environment. 
+Each script described below can be modified to suits your needs. For instance, you can:
+- Add specific tags to your instances.
+- Modify the Spark integration configuration.
 
-2. Copy and run the contents into a notebook. The notebook creates an init script that installs a Datadog Agent on your clusters.
-    The notebook only needs to be run once to save the script as a global configuration. 
-    - Set `<init-script-folder>` path to where you want your init scripts to be saved in.
 <!-- partial
 {{% site-region region="us,us3,us5,eu,gov,ap1" %}}
-
-3. Configure a new Databricks cluster with the cluster-scoped init script path using the UI, Databricks CLI, or invoking the Clusters API.
-    - Set the `DD_API_KEY` environment variable in the cluster's Advanced Options with your Datadog API key.
-    - Add `DD_ENV` environment variable under Advanced Options to add a global environment tag to better identify your clusters.
-    - Set `DD_SITE` to your site: {{< region-param key="dd_site" code="true" >}}.
+You can also define or modify environment variables with the cluster-scoped init script path using the UI, Databricks CLI, or invoking the Clusters API:
+    - The `DD_API_KEY` environment variable to better identify your clusters.
+    - The `DD_ENV` environment variable to better identify your clusters.
+    - Set `DD_SITE` to your site: {{< region-param key="dd_site" code="true" >}}. Defaults to `datadoghq.com`
 {{% /site-region %}}
 partial -->
 
+<div class="alert alert-warning">For security reasons, it's not recommended to define the `DD_API_KEY` environment in plain text directly in the UI. Instead, please use <a href="https://docs.databricks.com/en/security/secrets/index.html">Databricks secrets</a>.</div>
+
 
 #### Standard cluster
```

Comment:
```suggestion
```
Can we delete this heading since you removed the parallel section "Job cluster"? That way we can promote all of the headings up a level too. I added those suggestions below.